### PR TITLE
[7.5.2 backport] Backported recent changes from 8.x. Including "Writing custom shaders" and the rewritten post-processing and Volumes pages.

### PR DESCRIPTION
**Pull request by oleks-k**
Backported recent changes from 8.x. Including "Writing custom shaders" and the rewritten post-processing and Volumes pages.

### CODE REVIEW

File: com.unity.render-pipelines.universal/Documentation~/InstallingAndConfiguringURP.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ To use the Universal Render Pipeline (URP), you can start a new Project or upgra
...
 - [Install URP into an existing Unity Project](InstallURPIntoAProject.md). If you have started a Project using the Built-in Render Pipeline, you can install URP and configure your Project to use URP. When you do this, you must configure URP yourself. You will need to manually convert or recreate parts of your Project (such as lit shaders or post-processing effects) to be compatible with URP.
 
-**Note:** URP's integrated post-processing solution does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP's integrated post-processing solution. Custom post-processing effects will be supported in a forthcoming release of URP.
+**Note:** URP does not currently support custom post-processing effects. If your Project uses custom post-processing effects, these cannot currently be recreated in URP. Custom post-processing effects will be supported in a forthcoming release of URP.
```
Comment (phi-lira) [inline com.unity.render-pipelines.universal/Documentation~/InstallingAndConfiguringURP.md:8]: It's supported if used in PPv2. It's only the new built-in stack that doesn't support it yet.

**Review by phi-lira — APPROVED** [review]
Minor comment.

File: com.unity.render-pipelines.universal/Documentation~/InstallingAndConfiguringURP.md (hunk 1 above)
Comment (oleks-k) [reply com.unity.render-pipelines.universal/Documentation~/InstallingAndConfiguringURP.md]: Thanks, I've edited this paragraph in the new commit.

**Review by ellioman — APPROVED** [review]

**Comment by ellioman** [comment]
This is just a doc change so Yamato run is not required

**Pull request merged**